code/frameworks/nextjs/src/swc/loader.ts

```diff
@@ -31,7 +31,6 @@ export const configureSWCLoader = async (
     test: /\.((c|m)?(j|t)sx?)$/,
     include: [getProjectRoot()],
     exclude: [/(node_modules)/, ...Object.keys(virtualModules)],
-    enforce: 'post',
```

Comment:
now that the order of the loader is changed, are there potential side effects to this?

Reply:
I have adjusted the loader so that it behaves like the Babel loader. There might be side effects, but I am aware of none right now.